### ISSUE TITLE
Docker container monitor says restating containers are up

### ISSUE THREAD

**Issue by 404invalid-user** [louislam/uptime-kuma#5369]
### 📑 I have found these related issues/pull requests

found nothing

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

After setting up a Docker container monitor for my containers I have noticed when my containers crash and go into a restarting loop uptime-kuma will say they are still up

### 👟 Reproduction steps

add the docker socket to umptime-kuma

setup a docker container that has restart-always and then starts and crashes so state is constantly starting if you use `docker ps`

setup a monitor for that container in uptime-kuma

see that is is always up yet the container isn't actually up.

### 👀 Expected behavior

For uptime kuma to say the container is down and send a notification

### 😓 Actual Behavior

the monitor status continues to say up and I do not get a notification

### 🐻 Uptime-Kuma Version

1.23.15

### 💻 Operating System and Arch

ubuntu 22.04

### 🌐 Browser

google chrome, firefox

### 🖥️ Deployment Environment

- Runtime: Docker
- Database: sqlite
- Filesystem used to store the database on: SSD
- number of monitors: 4

**Comment by karimroshan** [comment]
I am also experiencing this bug. Any update on this?

**Comment by Tooa** [comment]
Experienced the same problem today.

I did a little bit of research. There is a similar problem in #4369 that is being addressed in !4372.

Unfortunately, there is no pull request for the issue described here.

**Comment by ezeholz** [comment]
More than a year of this, any development on this matter?
Thanks!

**Comment by CommanderStorm** [comment]
We merged a bunch of changes to the docker monitors internal behaviour. i am not sure which part is still relevant..

**Comment by CommanderStorm** [comment]
currently, we are pending when we are restarting, so not "up"

**Comment by ezeholz** [comment]
In pending, I did not receive a notification either, and the container was restarting indefinitely. Shouldn't we mark them as "down" completely?

**Comment by CommanderStorm** [comment]
There are a number of remaining issues, such as:
- https://github.com/louislam/uptime-kuma/issues/6025
- https://github.com/louislam/uptime-kuma/issues/6585

Generally, this needs someone investing time into this monitor, I think current default is likely fine so won't invest time myself.
Being in pending is better than being up.
So not a bug anymore.

**Comment by CommanderStorm** [comment]
Obviously, there could be more configuration, but that is a different issue that also would need someone willing to impelement the toggle and add a testcase for it.
